modules/runtime_modules.ts

```diff
@@ -0,0 +1,53 @@
+import { Message } from 'discord.js'
+import { loadModule, unloadModule, availableModules } from '../scepter'
+
+const loadModules = async (message: Message, args: string[]) => {
+  args.forEach(async (module: string) => {
+    if (!availableModules.includes(module)) {
+      await message.channel.send(`Module ${module} does not exist!`)
+      return
+    }
+
+    if (Object.keys(message.client['loadedModules']).includes(module)) {
+      await message.channel.send(`Module ${module} is already loaded. Not doing anything.`)
+      return
+    }
+
+    loadModule(module)
+    await message.channel.send(`Module ${module} is now available.`)
+  })
+}
+
+const unloadModules = async (message: Message, args: string[]) => {
+  args.forEach(async (module: string) => {
+    if (!availableModules.includes(module)) {
+      await message.channel.send(`Module ${module} does not exist!`)
+      return
+    }
+
+    if (!Object.keys(message.client['loadedModules']).includes(module)) {
+      await message.channel.send(`Module ${module} is already unloaded. Not doing anything.`)
+      return
+    }
+    unloadModule(module)
+    await message.channel.send(`Module ${module} has been unloaded.`)
+  })
+}
+
+export const name = 'runtime modules'
+export const commands = [
```

Comment:
These should have a minimum permission level

Reply:
What should it be? Mods or admins?

Reply:
Since it affects the entire bot, bot owner. This should probably accept multiple values in the future, though.

https://github.com/nixnest/scepter/blob/master/scepter.ts#L153